tests/ui/UnreadIndicatorsTest.js

```diff
@@ -184,7 +184,7 @@ function signInAndGetAppWithUnreadChat() {
 }
 
 describe('Unread Indicators', () => {
-    afterEach(Onyx.clear);
+    afterEach(() => Onyx.clear());
```

Comment:
Oh yeah - when there's no defined argument the default param didn't kick in because the keysToPreserve param ended up being a function:
![Screenshot 2022-12-12 at 11 34 35 AM](https://user-images.githubusercontent.com/4741899/207137934-11cebf86-d76b-4127-b8ca-10d7aa3bbc72.png)

Reply:
Ah ok weird, thanks!